zerberus/voice.cpp

```diff
@@ -121,7 +121,7 @@ void Voice::start(Channel* c, int key, int v, const Zone* z)
 
       phase.set(0);
       float sr = float(s->sampleRate()) / _zerberus->sampleRate();
-      phaseIncr.set(_zerberus->ct2hz(key * 100.0 + z->tune) * sr/_zerberus->ct2hz(z->keyBase * 100.0));
+      phaseIncr.set(_zerberus->ct2hz((((key - z->keyBase) * (z->pitchKeytrack / (double) 100.0)) + z->keyBase) * 100.0 + z->tune) * sr/_zerberus->ct2hz(z->keyBase * 100.0));
```

Comment:
This line would benefit to be exploded and made more understandable.
For example, this is a lot more understandable https://github.com/stevefolta/SFZero/blob/master/Source/SFZVoice.cpp#L291